config/deployments/cluster/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ kind: Kustomization
 # Do not set namespace here!
 
 resources:
-  - github.com/k8ssandra/cass-operator/config/deployments/cluster?ref=v1.8.0
+  - github.com/k8ssandra/cass-operator/config/deployments/cluster?ref=master
```

Comment:
Commit SHAs do work. It would look like this:

```
- github.com/k8ssandra/cass-operator/config/deployments/default?ref=8cb917829b7250278c5202ffcd16378d386599a9
```
I believe you need to specify the full SHA.

Reply:
Ah right, full SHAs do work indeed.